articles/native-platforms/android/09-mfa.md

```diff
@@ -0,0 +1,36 @@
+---
+title: MFA
+description: This tutorial will show you how to configure Multi Factor Authentication (MFA) via Google Authenticator in your app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+### 1. Enable MFA in your account
```

Comment:
@tenmaster Before this paragraph we need an intro text explaining what MFA is. You can copy and paste most of it from another quickstart, for example [angular2](https://auth0.com/docs/quickstart/spa/react/09-mfa)